src/en/general-development/setup/git-for-the-ss14-developer.md

```diff
@@ -449,6 +449,10 @@ Now, the fun part. We'll go to GitHub now and make a pull request for our funny
 
 Add a description, a nice title, some screenshots, and hopefully it gets merged.
 
+```admonish warning title="Don't do it now"
+Your pull request will be made directly in the official repository and will be visible to others. Ensure that your changes are thoroughly reviewed and ready for official evaluation. If you only want to test, create the PR in your own fork to avoid making it publicly visible.
```

Comment:
By default forks of spacestation 14 created on github are already publicly visible: [see this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-permissions-and-visibility-of-forks#about-visibility-of-forks). But it is correct that substantially more visibility will be received by creating said PR. 

This warning is valid, but it should indicate that any PR made to the upstream repository can be reviewed and commented on by any person.

Reply:
You're absolutely right. My original wording wasn't clear enough. Thank you for the feedback!
I've rewritten it, what do you think of this version?